.devcontainer/dev_install

```diff
@@ -18,4 +18,4 @@ pre-commit install
 cd db_migration
 
 # Check if the database has been initialized and if not initialize it
-alembic -c alembic.ini upgrade head
+# alembic -c alembic.ini upgrade head
```

Comment:
We should probably add database versioning to this microservice

Reply:
Will do this in a seperate PR

Reply:
👍